statuses.py

```diff
@@ -65,4 +65,8 @@
    "A hug in time saves nine 🙂.",
    "Hey you! I'm talking to you, yes you! I'm sending you a hug 🤗.",
    "Roses are red,\nViolets are blue,\nYou're so great,\nI just want to hug you!",
+    "A hug is like a boomerang – you get it back right away.",
+   "Hug while you can!",
+   "A hug is always the right size. —Winnie the Pooh",
```

Comment:
Maybe put the attribution to Winnie the Pooh in its own line (using a `\n` newline character)